README.md

```diff
@@ -1,5 +1,8 @@
 # The Tracing Simulator
 
+## Requirements
+Install graphviz (https://graphviz.org/download/)
```

Comment:
Why do we need this?

Reply:
This allows us to create pdfs that have representations of the graph created by the simulator.  I thought it would be good for debugging/ease of use, because it might be hard to visualize the graph when you just say "add_node" or "add_edge"